bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java

```diff
@@ -87,18 +111,33 @@ public KeyValueStorageRocksDB(String basePath, String subPath, DbConfigType dbCo
         this.emptyBatch = new WriteBatch();
 
         String dbFilePath = "";
+        if (dbConfigType == DbConfigType.EntryLocation) {
+            dbFilePath = conf.getEntryLocationRocksdbConf();
+        } else if (dbConfigType == DbConfigType.LedgerMetadata) {
+            dbFilePath = conf.getLedgerMetadataRocksdbConf();
+        } else {
+            dbFilePath = conf.getDefaultRocksDBConf();
+        }
+        if (Paths.get(dbFilePath).toFile().exists()) {
+            db = initializeRocksDBWithConfFile(basePath, subPath, dbConfigType, conf, readOnly, dbFilePath);
+        } else {
+            db = initializeRocksDBWithBookieConf(basePath, subPath, dbConfigType, conf, readOnly);
```

Comment:
here we log that we have't found the file and that we will read the configuration from the main bookkeeper configuration